packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -185,6 +189,12 @@ marketVolConfigurator18 = "<%= settings.storkExecutor18 %>"
 marketVolConfigurator19 = "<%= settings.storkExecutor19 %>"
 marketVolConfigurator20 = "<%= settings.storkExecutor20 %>"
 
+# TODO: add proper addresses
+[var.share_token_roles]
+rseliniSubscriber1 = "0x0000000000000000000000000000000000000000"
```

Comment:
hmm, let's just put 556 as the subscriber/redeemer or alternatively (probs better) feel free to just create a new eoa for this, will also be used in the rebalancing script

Reply:
just created new one